src/InlineFilesTransformer.ts

```diff
@@ -81,14 +81,8 @@ export function factory(cs: ConfigSet) {
    * Traverses the AST down to the relevant assignments anywhere in the file
    * and returns a boolean indicating if it should be transformed.
    */
-  function isPropertyAssignmentToTransform(
-    node: Node
-  ): node is PropertyAssignment {
-    return (
-      ts.isPropertyAssignment(node) &&
-      ts.isIdentifier((node as PropertyAssignment).name) &&
-      TRANSFORM_PROPS.includes(((node as PropertyAssignment).name as Identifier).text)
-    );
+  function isPropertyAssignmentToTransform(node: Node): node is PropertyAssignment {
+    return ts.isPropertyAssignment(node) && ts.isIdentifier(node.name) && TRANSFORM_PROPS.includes(node.name.text);
```

Comment:
Interesting, I was getting errors without the casts

Reply:
weird, I tested with both windows and macos, `yarn build` is fine for me. Maybe you need to remove `node_modules` and reinstall 😃

Reply:
Maybe it was still on an older TS version with less inference when I originally implemented this.